sql/core/src/test/resources/sql-tests/inputs/group-by-filter.sql

```diff
@@ -1,5 +1,8 @@
 -- Test filter clause for aggregate expression.
 
+--CONFIG_DIM1 spark.sql.optimizeNullAwareAntiJoin=true
+--CONFIG_DIM1 spark.sql.optimizeNullAwareAntiJoin=false
+
```

Comment:
Thanks for adding these. It gives us more confidence.

Reply:
thanks to @cloud-fan I know of this better way to do e2e case coverage when adding a new feature.